README.md

```diff
@@ -1,11 +1,18 @@
 # Chimney <img src="chimney.png" alt="Chimney logo" width="64" />
 
 ![CI build](https://github.com/scalalandio/chimney/workflows/CI%20build/badge.svg)
-[![Maven Central](https://img.shields.io/maven-central/v/io.scalaland/chimney_2.12.svg)](http://search.maven.org/#search%7Cga%7C1%7Cchimney)
-[![Javadocs](https://www.javadoc.io/badge/io.scalaland/chimney_2.13.svg?color=red&label=scaladoc)](https://www.javadoc.io/doc/io.scalaland/chimney_2.13/latest/io/scalaland/chimney/index.html)
 [![codecov.io](http://codecov.io/github/scalalandio/chimney/coverage.svg?branch=master)](http://codecov.io/github/scalalandio/chimney?branch=master)
 [![License](http://img.shields.io/:license-Apache%202-green.svg)](http://www.apache.org/licenses/LICENSE-2.0.txt) [![Join the chat at https://gitter.im/scalalandio/chimney](https://badges.gitter.im/scalalandio/chimney.svg)](https://gitter.im/scalalandio/chimney?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[![Scala.js](https://www.scala-js.org/assets/badges/scalajs-1.0.0.svg)](https://www.scala-js.org)
+
+[![Chimney JVM versions](https://index.scala-lang.org/scalalandio/chimney/chimney/latest-by-scala-version.svg?platform=jvm)](https://search.maven.org/artifact/io.scalaland/chimney_2.13) <br>
+[![Chimney Scala.js 1.x versions](https://index.scala-lang.org/scalalandio/chimney/chimney/latest-by-scala-version.svg?platform=sjs1)](https://search.maven.org/artifact/io.scalaland/chimney_sjs1_2.13) <br>
+[![Chimney Scala.js 0.6 versions](https://index.scala-lang.org/scalalandio/chimney/chimney/latest-by-scala-version.svg?platform=sjs0.6)](https://search.maven.org/artifact/io.scalaland/chimney_sjs0.6_2.13) <br>
```

Comment:
Maybe no need for sjs 0.6, as is not supported any more?

Reply:
I was thinking about leaving SJS 0.6, SN 0.3 and Scala 2.11 to show the last available historical versions on 1 glance for people who for some reason got stuck at them.

But I also stated that currently supported are only 2.12, 2.13 for JVM, SJS 1.x and Native 0.4 (the last one since this PR).

Reply:
Historical versions are easily discoverable on Maven Central, I wouldn't optimize main welcome section for `people who for some reason got stuck at` the old versions, but focus on having just few most important badges.

Alternatively, you can create a separate section in the readme with the table specifying which versions for which platforms are available (assuming we won't have to maintain the version numbers there manually).

Reply:
I left 1 version in the header and a full matrix in getting started section with SBT example.